cmd/lotus/daemon.go

```diff
@@ -574,3 +603,122 @@ func ImportChain(ctx context.Context, r repo.Repo, fname string, snapshot bool)
 
 	return nil
 }
+
+func slashConsensus(a lapi.FullNode, p string, from string) error {
```

Comment:
This really doesn't belong in this file. Should probably be a separate DI module, enabled with config, not a random flag on one of the main commands.